### PR TITLE
change README

### PR DESCRIPTION
**Pull request by Pantyhose-X**
Add dragonfireclient description

### CODE REVIEW

File: Minetest.md [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+- A specific file can be specified on the command line:
+    `--config <path-to-file>`
+- A run-in-place build will look for the configuration file in
+    `location_of_exe/../minetest.conf` and also `location_of_exe/../../minetest.conf`
+
+Command-line options
+--------------------
+- Use `--help`
+
+Compiling
```
Comment (corarona) [inline Minetest.md:129]: This part has been changed to document the process for dragonfire. It should stay in readme.md

**Review by corarona — CHANGES_REQUESTED** [review]
Looks good but please copy the compiling/installation section back to readme

**Comment by corarona** [comment]
This is where it was added: https://github.com/dragonfireclient/dragonfireclient/pull/35

**Comment by Pantyhose-X** [comment]
Already changed, please merge it

**Pull request closed without merging**